common/src/main/java/org/jboss/hal/testsuite/page/runtime/DeploymentPage.java

```diff
@@ -16,11 +25,59 @@
 public class DeploymentPage extends ConfigurationPage {
 
     private static final By CONTENT = By.className(PropUtils.get("page.content.gwt-layoutpanel"));
+    private static final String CONTENT_TABLE_ROW = "table:contains('Data') tr.form-attribute-row";
 
     public StandaloneDeploymentsArea getDeploymentContent() {
         WebElement content =  getContentRoot().findElement(CONTENT);
 
         return Graphene.createPageFragment(StandaloneDeploymentsArea.class, content);
     }
 
+    public String getDeploymentContentTableRow(String rowContains) {
+        StringBuffer selector = new StringBuffer(CONTENT_TABLE_ROW);
+        if (rowContains != null && !rowContains.isEmpty()) {
+            selector.append(":contains('" + rowContains + "')");
+        } else {
+            return "";
+        }
+
+        WebElement content = getContentRoot().findElement(new ByJQuery(selector.toString()));
+        return content.getText();
+    }
+
+    public List<String> getDeploymentBrowsedContentItems() {
+        By selector = ByJQuery.selector("table.browse-content tr");
+        List<String> items = new ArrayList<String>();
+
+        List<WebElement> webElements = getContentRoot().findElements(selector);
+        for (WebElement webElement : webElements ) {
+            items.add(webElement.getText());
+        }
+
+        return items;
+    }
+
+    public void viewDeployment(String deploymentName) {
```

Comment:
Please add javadoc to this method. Purpose of this method - to open deployment detail - may not be obvious.